### ISSUE TITLE
Callable parameter usage false positive for callable objects

### ISSUE THREAD

**Issue by Jean85** [kalessil/phpinspectionsea#1115]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug/False-Positive                          |
| Plugin         | Php Inspections (EA Extended) 3.0.8.1  |
| Language level | PHP 7.1 |

Current behaviour (description/screenshot:)
--
```php
    function foo(callable $callable)
    {
        if (\is_object($callable) && \method_exists($callable, '__invoke')) {
            $reflection = new \ReflectionMethod($callable, '__invoke');
            $class = $reflection->getDeclaringClass();
        }
        // ...
    }
```
![image](https://user-images.githubusercontent.com/6729988/48936670-ed513100-ef0b-11e8-9fea-0d2081ef7080.png)

Expected behaviour
--
No warning, since `Closure` (or any other object implementing `__invoke`) is a valid `callable`.

**Comment by kalessil** [comment]
Oh, good catch @Jean85! I'll take care of the issue.

**Comment by kalessil** [comment]
Fixed!
